.forge-snapshots/positionDescriptor bytecode size.snap

```diff
@@ -1 +1 @@
-24177
\ No newline at end of file
+24108
```

Comment:
this gas snapshot is all over the place 😓

Reply:
Yeah.. I've been noticing that. Unsure why